force/sobjects.go

```diff
@@ -112,6 +114,41 @@ func (forceApi *ForceApi) DeleteSObject(id string, in SObject) (err error) {
 	return
 }
 
+func idsFromURIs(uris []string) []string {
+	ids := make([]string, len(uris))
+	for i, uri := range uris {
+		parts := strings.Split(uri, "/")
+		ids[i] = parts[len(parts)-1]
+	}
+	return ids
+}
+
+func (forceApi *ForceApi) GetSFIDsByExternalId(apiName, externalKey, externalId string) ([]string, int, error) {
+	uri := fmt.Sprintf("%v/%v/%v", forceApi.apiSObjects[apiName].URLs[sObjectKey], externalKey, externalId)
+
+	params := url.Values{"fields": []string{"Id"}}
+	sobj := sobjects.BaseSObject{}
+
+	statusCode, err := forceApi.Get(uri, params, &sobj)
```

Comment:
Can we separate out each Get call into its own mini function? i.e. GetSingle vs GetMultiple or something to reflect the different semantics.